build.sbt

```diff
@@ -33,6 +33,7 @@ lazy val core = crossProject(JVMPlatform, JSPlatform)
     libraryDependencies ++= Seq(
       "org.typelevel" %%% "cats-core" % "2.7.0",
       "org.typelevel" %%% "cats-effect" % "3.3.13",
+      "co.fs2" %%% "fs2-core" % "3.2.11",
```

Comment:
If you only need `ByteVector` you can include scodec-bits directly. I just mentioned its in fs2 as a proof of stability/wide adoption 😉

Reply:
Highly likely we will need fs2 in the classpath to provide a proper tracing of streams.